distribution/lib/Standard/Database/0.0.0-dev/src/Data/Table.enso

```diff
@@ -1084,17 +1094,30 @@ type Table
          name will be in use, in this case it will be resolved using the normal
          renaming strategy - adding subsequent `_1`, `_2` etc.
 
-       ? Result Ordering
+       ? Row Ordering For In-Memory Tables
 
-         Rows in the result are first ordered by the order of the corresponding
-         rows from the left table and then the order of rows from the right
-         table. This applies only if the order of the rows was specified (for
-         example, by sorting the table; in-memory tables will keep the memory
-         layout order while for database tables the order may be unspecified).
+         This operation requires a well-defined order of rows in the input
+         tables. In-memory tables rely on the ordering stemming directly from
+         their layout in memory. Database tables may not impose a deterministic
+         ordering. If the table defines a primary key, it is used to by default
+         to ensure deterministic ordering. That can be overridden by specifying
+         a different ordering using `Table.order_by`. If no primary key was
+         defined nor any ordering was specified explicitly by the user, the
+         order of columns is undefined and the operation will fail, reporting a
+         `Undefined_Column_Order` problem and returning an empty table.
+
+       ? Result Ordering For Database Tables
```

Comment:
Let's be consistent - it's either `Row Ordering` or `Result Ordering`. Both seem fine (I prefer `Row`), but both sections should be named consistently.

Reply:
Changed to Row and made them the same.